spec/bifrost/topic_spec.rb

```diff
@@ -15,6 +15,11 @@
     expect { invalid_topic.save }.to raise_error(TypeError)
   end
 
+  it 'should not accept names with spaces in them' do
+    invalid_topic = Bifrost::Topic.new('topic name')
+    expect { invalid_topic.save }.to raise_error(URI::InvalidURIError)
```

Comment:
@lfportal I think we should wrap this. URI::InvalidURIError makes no sense in the context of the Bifrost